packages/docgen/README.md

```diff
@@ -33,9 +33,9 @@ This command will generate a file named `entry-point-api.md` containing all the
 * **--output** `(String)`: Output file that will contain the API documentation.
 * **--to-section** `(String)`: Append generated documentation to this section in the Markdown output. To be used by the default Markdown formatter. Depends on `--output` and bypasses the custom `--formatter` passed, if any.
 * **--to-token**: Embed generated documentation within the start and end tokens in the Markdown output. To be used by the default Markdown formatter.Depends on `--output` and bypasses the custom `--formatter` passed, if any.
-  * Start token: `<!-- START TOKEN(Autogenerated API docs) -->`
+  * Start token: <code>&lt;!-- START TOKEN(Autogenerated API docs) --></code>
```

Comment:
Why is this necessary (and not in the item list just below this one)?

Reply:
I recall in initial implementations that I was seeing that docgen was trying to insert documentation here 😆 Without digging further, it's not clear exactly why it wouldn't be an issue for the other instance of token pairs.

Reply:
Ah! I now see how this file is matched by the script: when you run the script without any files as input, it'll search the whole codebase for candidates. Made a few tweaks at https://github.com/WordPress/gutenberg/pull/18840/commits/4119f471d3b2ab222420eb67a9b74a5fa05fc3e1 for consistency.